opencti-platform/opencti-front/src/utils/filters/filtersUtils.tsx

```diff
@@ -262,6 +263,7 @@ export const findFilterIndexFromKey = (
   return null;
 };
 
+// add entity types context in filters
 export const filtersWithEntityType = (
```

Comment:
rename function to a verb for consistency, for instance

```suggestion
export const injectEntityTypeFilterInFilterGroup = (
```

Reply:
done